test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetListCommandTest.cs

```diff
@@ -101,6 +101,147 @@ public void ListCommand_WithUserSpecifiedSource()
             }
         }
 
+        // Skip: https://github.com/NuGet/Home/issues/11576
+        [PlatformTheory(Platform.Windows, Platform.Linux, SkipMono = true)]
+        [InlineData(false, false, false)]
+        [InlineData(false, false, true)]
+        [InlineData(false, true, false)]
+        [InlineData(false, true, true)]
+        [InlineData(true, false, false)]
+        [InlineData(true, false, true)]
+        [InlineData(true, true, false)]
+        [InlineData(true, true, true)]
+        public void ListCommand_WithLocalSource_WithMatchText_ReturnLatestVersionOnlyPerPackageId(bool includeMatchingText, bool includeListed, bool preRelease)
```

Comment:
"listed" is a concept that only exists in HTTP feeds, not local folder feeds. This is making the test matrix twice as large without benefit.

Reply:
Sure, removed.